app/mailers/notifier_mailer.rb

```diff
@@ -81,10 +81,18 @@ def notify_new_finding_answer(users, finding_answer)
     mail to: users_to_notify_for(users).map(&:email),
          subject: prefix.upcase + t(
            'notifier.notify_new_finding_answer.title',
-           review: finding_answer.finding.review.to_s
+           review:     finding_answer.finding.review.to_s,
+           finding_id: finding_answer.finding.id
          )
   end
 
+  def notify_not_added_answer(email_to, answer)
```

Comment:
```suggestion
  def notify_action_not_found(email_to, answer)
```
Le pondría algo así, ya que lo usaríamos para los casos en que no encontramos una acción que realizar en base al subject.

Reply:
me parece perfecto